src/main/java/org/broadinstitute/consent/http/db/DatasetDAO.java

```diff
@@ -102,17 +102,26 @@
     @SqlUpdate("UPDATE dataset SET update_date = :updateDate, update_user_id = :updateUserId WHERE datasetid = :datasetId")
     void updateDatasetUpdateUserAndDate(@Bind("datasetId") Integer datasetId, @Bind("updateDate") Timestamp updateDate, @Bind("updateUserId") Integer updateUserId);
 
-    @UseRowMapper(DataSetPropertiesMapper.class)
+    @UseRowMapper(DataSetMapper.class)
```

Comment:
While this mapper class is probably the wrong approach, it isn't necessary to use the annotation here since the class uses it as a default on line 34, `@RegisterRowMapper(DataSetMapper.class)`

Reply:
`DataSetPropertiesMapper` used a hashmap strategy to condense rows down to a single record, which `DataSetMapper` does not do.